ui/raidboss/data/06-ew/dungeon/another_sildihn_subterrane.ts

```diff
@@ -126,22 +126,31 @@ const triggerSet: TriggerSet<Data> = {
       response: Responses.goRight(),
     },
     {
-      id: 'ASS Suds Collect',
-      // 7757 Bracing Suds (Wind / Donut)
-      // 7758 Chilling Suds (Ice / Cardinal)
-      // 7759 Fizzling Suds (Lightning / Intercardinal)
-      type: 'StartsUsing',
-      netRegex: { id: ['7757', '7758', '7759'], source: 'Silkie' },
-      run: (data, matches) => data.suds = matches.id,
+      id: 'ASS Suds Gain',
+      // CE1 Bracing Suds (Wind / Donut)
+      // CE2 Chilling Suds (Ice / Cardinal)
+      // CE3 Fizzling Suds (Lightning / Intercardinal)
+      type: 'GainsEffect',
+      netRegex: { effectId: 'CE[1-3]', target: 'Silkie' },
+      run: (data, matches) => data.suds = matches.effectId,
+    },
+    {
+      id: 'ASS Suds Lose',
+      // CE1 Bracing Suds (Wind / Donut)
+      // CE2 Chilling Suds (Ice / Cardinal)
+      // CE3 Fizzling Suds (Lightning / Intercardinal)
+      type: 'LosesEffect',
+      netRegex: { effectId: 'CE[1-3]', target: 'Silkie', capture: false },
+      run: (data, _matches) => delete data.suds,
```

Comment:
Can shorten this to:
`      run: (data) => delete data.suds,`

Reply:
Here's a suggestion for easier committing

```suggestion
      run: (data) => delete data.suds,
```